internal/usage/usage.go

```diff
@@ -371,4 +371,8 @@ dbName and collection are only required for built-in roles.`
 	OplogSizeMB                               = "Storage limit of cluster's oplog expressed in megabytes."
 	SampleRefreshIntervalBIConnector          = "Interval in seconds at which the mongosqld process re-samples data to create its relational schema."
 	SampleSizeBIConnector                     = "Number of documents per database to sample when gathering schema information."
+	ExporterClusterName                       = "One or more comma separated cluster names to import"
+	//#nosec
+	ExporterIncludeSecrets  = "Generate kubernetes secrets with data for projects, users, deployments entities"
+	ExporterTargetNamespace = "Namespaces to use for generated kubernetes entities"
```

Comment:
Could you follow a similar approach as [here?](https://github.com/mongodb/mongodb-atlas-cli/blob/9b51dd955e8b8a66340d1145ea00929b041ed561/internal/usage/usage.go#L242)

`	LinkTokenAccessListCIDREntries            = "IP address access list entries that are associated with the link-token." //nolint:gosec // This is just a message not a credential`

Reply:
Fixed